python/ray/train/callbacks/logging.py

```diff
@@ -20,7 +20,7 @@
 
 
 class TrainingLogdirMixin:
-    def start_training(self, logdir: str):
+    def start_training(self, logdir: str, **info):
```

Comment:
We don't need info here right?

Reply:
see https://github.com/ray-project/ray/pull/21357#issuecomment-1005070920